swift-api-examples/SherpaOnnx.swift

```diff
@@ -920,6 +920,7 @@ func sherpaOnnxOfflineTtsModelConfig(
   vits: SherpaOnnxOfflineTtsVitsModelConfig = sherpaOnnxOfflineTtsVitsModelConfig(),
   matcha: SherpaOnnxOfflineTtsMatchaModelConfig = sherpaOnnxOfflineTtsMatchaModelConfig(),
   kokoro: SherpaOnnxOfflineTtsKokoroModelConfig = sherpaOnnxOfflineTtsKokoroModelConfig(),
+  zipvoice: SherpaOnnxOfflineTtsZipvoiceModelConfig = SherpaOnnxOfflineTtsZipvoiceModelConfig(),
```

Comment:
Can you follow our convention to add a function
```
sherpaOnnxOfflineTtsZipvoiceModelConfig
```

Note it starts with the lowercase `s`.